README.md

```diff
@@ -29,23 +29,145 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
 
 ## Motivation
 
-* Users need all types of duration format depending on the requirements of their application. For example, in the airlines websites, to show how long a flight takes, the duration should be in Short or Narrow format
+* Users need all types of duration format depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
   - 1 hr 40 min 60 sec → Short 
   - 1 h 40 m 60 sec  → Narrow
 
-* Furthermore, for TTS (text to speech) applications, Wide format is needed
-  - 1 hour, 40 minutes and 60 seconds.
-
+## Requirements & Design
```

Comment:
I'm not sure I've seen such a section in an explainer before, but I have to admit that it seems useful. Perhaps we should move it into it's own file called `requirements.md` or something?

Reply:
I think it's fine to keep in the README, especially at Stage 2.

Reply:
Okay, I will leave it for now, and we can discuss updating it before Stage 3.